js/login.js

```diff
@@ -0,0 +1,41 @@
+/* variables */
+
+let logIn = document.getElementById("login-btn");
+let loginForm = document.querySelector(".login-form");
+
+logIn.addEventListener("click", (e) => {
+    e.preventDefault();
+
+
+    let emailId = document.getElementById('email').value;
+    let userPwd = document.getElementById('password').value;
+    const userData = JSON.parse(localStorage.getItem('userData')) || [];
+  
+
+
+let keyProp = Object.keys(userData);
+
+let propertyValue = Object.values(userData);
+
+let email, passwd;
+let found = "n";
+
+for(let i = 0; i < keyProp.length; i++){ 
+        email = propertyValue[i].userEmail;
+        passwd = propertyValue[i].password;
+        console.log(email);
+        console.log(passwd);
```

Comment:
```suggestion
function checkCredentials(emailId, userPwd, userData) {
  for (const user of userData) {
    if (user.userEmail === emailID && user.password === userPwd) {
      return true;
    }
  }
  return false;
}
```
We can use a function to verify user credentials. It takes three parameters: emailId, userPwd, and userData. The function will iterate through the userData to check if there is a user with a matching email and password. If a match is found, the function will return true.